bigquery_etl/view/publish_views.py

```diff
@@ -23,6 +23,11 @@
     # Dataset glam-fenix-dev:glam_etl was not found
     # TODO: this should be removed if views are to be automatically deployed
     *[str(path) for path in Path("sql/glam-fenix-dev").glob("glam_etl/**/view.sql")],
+    # Materialized views
+    "telemetry_derived/experiment_events_live_v1/view.sql",
+    "org_mozilla_firefox_derived/experiment_events_live_v1/view.sql",
+    "org_mozilla_firefox_beta_derived/experiment_events_live_v1/view.sql",
+    "org_mozilla_fenix_derived/experiment_events_live_v1/view.sql",
```

Comment:
Maybe we want to call these `script.sql` or `init.sql` instead of `view.sql`? I could imagine that in the future we would reconfigure `view.sql` files to contain just the query content and not the full `CREATE VIEW` statement, but we'd likely still need to have materialized views be full DDL definitions.

`init.sql` captures the idea that this is something that's run once.